org-cyf-sdc/content/tools/sprints/3/process_spelunking_workshop.md

```diff
@@ -0,0 +1,539 @@
+# Workshop: Process Spelunking
+
+This workshop will teach you about how the programs you write, interact with files.
+And in the process, we'll learn more about the kernel, and how it helps your programs do things.
+
+## Preparation
+
+### Linux
+
+In order to participate in this workshop, you will need a linux computer.
+I recommend using an ubuntu VM in AWS (free tier instance), or alternatively a VM inside virtual box.
```

Comment:
```suggestion
If you're in a CYF class, there are laptops you can borrow for this workshop - your volunteers should know how to access them.

Otherwise, I recommend using an ubuntu VM in AWS (free tier instance), or alternatively a VM inside virtual box.
```